### ISSUE TITLE
Primary group for user is changed to docker

### ISSUE THREAD

**Issue by ejsmit** [geerlingguy/ansible-role-docker_arm#8]
The current role currently uses the following to change the group membership:
```
  user:
    name: "{{ item }}"
    group: docker
    append: true
  with_items: "{{ docker_users }}"
```

which cause this to happen:
```
$ id
uid=1000(pi) gid=995(docker) groups=995(docker),4(adm),  ...
```

The `group`  should be `groups`.  This will leave the user in its original primary group (pi, gid 1000) and only add the user to the docker group.  

I also checked your docker role.  It correctly uses groups.

**Comment by geerlingguy** [comment]
You're correct, thanks for catching this oversight! I'll have this fixed ASAP.

**Comment by geerlingguy** [comment]
This is available in 3.1.0.
